helm/istio/charts/galley/templates/meshclusterrole.yaml

```diff
@@ -8,9 +8,6 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 rules:
-- apiGroups: ["admissionregistration.k8s.io"]
-  resources: ["validatingwebhookconfigurations"]
-  verbs: ["*"]
 # required to be able to set OwnerReference with BlockOwnerDeletion=true
```

Comment:
Can we remove the following too?  I thought we were going to get rid of the owner reference on the webhook.